cmd/tikv-server/Cargo.toml

```diff
@@ -28,6 +28,9 @@ test-engines-panic = ["server/test-engines-panic"]
 
 nortcheck = ["server/nortcheck"]
 
+pprof-dwarf = ["server/pprof-dwarf"]
+pprof-fp = ["server/pprof-fp"]
```

Comment:
Can we directly set `tikv/pprof-*` features here? The features in `server` crate don't seem necessary...

Reply:
@sticnarf Fixed, PTAL~